src/test/regress/expected/multi_subquery_complex_queries.out

```diff
@@ -683,6 +768,45 @@ FROM
       21 |     985
 (6 rows)
 
+-- nested lateral join at top most level
+SELECT "some_users_data".user_id, "some_recent_users".value_3
+FROM
+	(SELECT filter_users_1.user_id, value_3
+     FROM
+     	(SELECT "users"."user_id"
+         FROM users_table as "users"
+         WHERE user_id > 20 and user_id < 70 and users.value_2 = 200
+        ) filter_users_1
+        JOIN LATERAL
+        (SELECT user_id, value_3
+         FROM events_table as "events"
+         WHERE  user_id > 20 and user_id < 70
+           AND ("events".user_id = "filter_users_1".user_id)
+         ORDER BY value_3 DESC
+         LIMIT 1
+        ) "last_events_1" ON TRUE
+     ORDER BY value_3 DESC
+     LIMIT 10
+   ) "some_recent_users"
+   JOIN LATERAL
+   (SELECT "users".user_id
+    FROM users_table as "users"
+    WHERE "users"."user_id" = "some_recent_users"."user_id"
+        AND users.value_2 > 200
+    LIMIT 1
+   ) "some_users_data" ON TRUE
+ORDER BY value_3 DESC
+LIMIT 10;
```

Comment:
To ensure getting the same results per run, could you also add `user_id` to the ORDER BY? To the others as well?